microservices/realtime/pkg/utils/constants.go

```diff
@@ -0,0 +1,6 @@
+package utils
+
+// ROLES
+const ROLE_SUPER_ADMIN = "ROLE_SUPER_ADMIN"
+const ROLE_BRAND_ADMIN = "ROLE_BRAND_ADMIN"
+const ROLE_COMPANY_ADMIN = "ROLE_COMPANY_ADMIN"
```

Comment:
Are this constants used from more than one that package. If not, I will move it to the package that use them.
The other alternative is to move all constants here, like redis_status.go constants